services/notifications/pkg/channels/channels.go

```diff
@@ -3,9 +3,12 @@ package channels
 
 import (
 	"context"
+	"crypto/rand"
```

Comment:
I am not really sure if we need crypto save random numbers for the message ids. I guess a properly seeded pseudo random numbers from `math/crypto` could also be enough for that purpose.
But I guess it's fine this way as well.